pixel/head.html

```diff
@@ -1,6 +1,13 @@
 <script>
   (function() {
     var gtmId = "{{settings.gtmId}}";
+    var mergeUAEventsValue = "{{settings.mergeUAEvents}}"
+    var mergeUAEvents = mergeUAEventsValue == 'true'
```

Comment:
Would this work? This way, we could type `mergeUAEvents` as a boolean instead of a string in `react/typings/global.d.ts`.
```suggestion
    var mergeUAEvents = Boolean("{{settings.mergeUAEvents}}")
```

Reply:
This approach not work cause `Boolean('false') === true`, and the value stored in settings is parsed to string, even if the type declared in `react/typings/global.d.ts` be `boolean`
Example:
<img width="188" alt="Captura de Tela 2023-01-18 às 14 36 31" src="https://user-images.githubusercontent.com/36740164/213253982-490392ae-6b85-4fca-9b6b-68d8f227f750.png">

Reply:
I see. Thanks for clearing that up!